packages/paste-core/layout/stack/__tests__/stack.test.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {render as testRender} from '@testing-library/react';
+import {render as testRender, screen} from '@testing-library/react';
```

Comment:
Are you using `screen` as all?

Reply:
I don't think you need to re assign render as testRender here either

Reply:
Tried to use a `screen` method to test as a `header`, but it wasn't working as I thought. Just forgot to remove it.